### PR TITLE
Feature ge additional information

### PR DESCRIPTION
**Pull request by mihajlo-jaric**
Display and edit additional or basic information about objects.

### CODE REVIEW

File: WPFHospitalEditor/AdditionalInformation.xaml.cs [hunk 1]
```diff
@@ -0,0 +1,233 @@
...
+            Grid.SetColumn(name, 1);
+            
+            DynamicGrid.Children.Add(name);
+        }
+
+        private void createRows(string[] contentRows, Grid DynamicGrid)
+        {
+            RowDefinition gridRow1 = new RowDefinition();
+            gridRow1.Height = new GridLength(2);
+            DynamicGrid.RowDefinitions.Add(gridRow1);
```
Comment (rammba) [inline WPFHospitalEditor/AdditionalInformation.xaml.cs:197]: You can extract this in separate method.

File: WPFHospitalEditor/Building.xaml.cs [hunk 2]
```diff
@@ -143,5 +143,38 @@ private void addToLegend(Rectangle rectangle, TextBlock textblock)
...
+            MapObject chosenMapObject = checkWhichObjectIsClicked(e, buildingFloors[index].getAllFloorMapObjects());
+            if (chosenMapObject != null)
+            {
+                openAdditionalInformationDialog(chosenMapObject);
+            }
+        }
+        private MapObject checkWhichObjectIsClicked(MouseButtonEventArgs e, List<MapObject> allMapObjectsShowed)
+        {
+            for (int i = 0; i < allMapObjectsShowed.Count; i++)
+            {
```
Comment (rammba) [inline WPFHospitalEditor/Building.xaml.cs:159]: This method is same as one in HospitalMap class.

**Review by rammba — CHANGES_REQUESTED** [review]

File: WPFHospitalEditor/AdditionalInformation.xaml.cs [hunk 3]
```diff
@@ -0,0 +1,233 @@
...
+            var bc = new BrushConverter();
+            DynamicGrid.Background = (Brush)bc.ConvertFrom("#FFC6F5F8");
+
+            createColumns(DynamicGrid);
+
+            createRows(contentRows, DynamicGrid);
+
+            addMapObjectName(mapObject, bc, DynamicGrid);
+
+            createRowContent(contentRows, DynamicGrid);
```
Comment (hi-imAndy) [inline WPFHospitalEditor/AdditionalInformation.xaml.cs:75]: DynamicGrid object should be a field in class AdditionalInformation, in this way we will decrease number of parameters of these functions.

File: WPFHospitalEditor/AdditionalInformation.xaml.cs [hunk 4]
```diff
@@ -0,0 +1,233 @@
...
+            DynamicGrid.Children.Add(cancel);
+        }
+
+        private void insertData(Grid DynamicGrid, List<string> labelContent, List<string> value)
+        {
+            for (int i = 0; i < labelContent.Count; i++)
+            {
+                insertLabels(DynamicGrid, labelContent, i);
+                insertTextBoxes(DynamicGrid, value, i);
+            }
```
Comment (hi-imAndy) [inline WPFHospitalEditor/AdditionalInformation.xaml.cs:141]: labelContent is a field, so you don't need to pass it as a parameter

File: WPFHospitalEditor/AdditionalInformation.xaml.cs [hunk 5]
```diff
@@ -0,0 +1,233 @@
...
+                if (contentRows[i].Equals(""))
+                    break;
+
+                string[] label = contentRows[i].Split("=");
+                labelContent.Add(label[0]);
+                value.Add(label[1]);
+            }
+
+            insertData(DynamicGrid, labelContent, value);
+
```
Comment (hi-imAndy) [inline WPFHospitalEditor/AdditionalInformation.xaml.cs:93]: value is a field

File: WPFHospitalEditor/AdditionalInformation.xaml.cs (hunk 4 above)
Comment (hi-imAndy) [inline WPFHospitalEditor/AdditionalInformation.xaml.cs:141]: these functions should have singular names (insertLabel)

**Review by hi-imAndy — CHANGES_REQUESTED** [review]

**Review by rammba — APPROVED** [review]

**Comment by rammba** [comment]
[(0) PR Feature ge additional information](https://trello.com/c/GVgkB20m/73-0-pr-feature-ge-additional-information)

**Review by hi-imAndy — APPROVED** [review]

**Pull request merged**